src/pages/EventsPage.tsx

```
@@ -23,433 +26,300 @@ import {
                   />
-                  <div className="absolute inset-0 bg-gradient-to-t from-black/80 via-transparent to-transparent opacity-0 group-hover:opacity-100 transition-opacity flex items-end p-4">
-                    <span className="text-sm font-bold text-white line-clamp-2">{flyer.title.rendered}</span>
+                </div>
+              </div>
+
+              <div className="flex flex-col">
+                <div className="bg-primary/10 border border-primary/20 self-start px-4 py-1.5 rounded-full text-primary text-xs font-bold uppercase tracking-widest mb-6">
+                  {t('events_status_upcoming', 'Próximo Evento')}
+                </div>
+
+                <h1 className="text-4xl md:text-5xl font-black font-display mb-6" dangerouslySetInnerHTML={{ __html: title }} />
```

Biome (2.3.14)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)